src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -243,6 +243,23 @@ namespace Microsoft.FSharp.Collections
         [<CompiledName("Empty")>]
         val empty<'T> : seq<'T>
 
+        /// <summary>Produces the set difference of two sequences by using generic hash and equality comparisons to compare values.</summary>
```

Comment:
Ok. Then we should keep it that way. It's just a bit strange to talk about sets when we're actually in Streams or lists...